src/main/resources/static/themes/cyverse/css/cas-1-0-1.css

```diff
@@ -7563,33 +7649,32 @@ footer {
  * Add in our own custom social colors.
  */
 .btn-submit {
-  background-color: #28a745;
+  background-color: #0971ab;
   color: #fff;
-  background-color: #28a745;
   border-color: rgba(0, 0, 0, 0.2); }
   .btn-submit:focus, .btn-submit.focus {
     color: #fff;
-    background-color: #1e7e34;
+    background-color: #6cc1ec;
```

Comment:
The official CyVerse light-blue is `#99d9ea`,
 so I'm not sure if we want to use that in place of `#6cc1ec`.

Reply:
I'm not sure about this one either. I'll have to take a closer look at the CSS.

Reply:
I switched to the official CyVerse color in this case too.